### PR TITLE
Revert "#56, #57, #58"

### PR DESCRIPTION
**Pull request by akhramov**

### CODE REVIEW

**Comment by akhramov** [comment]
@bcc32 / @nshi can you review this one, please?

Side note, we apparently had CI on travis, but it broke

**Review by nshi — APPROVED** [review]

File: org-wild-notifier.el [hunk 1]
```diff
@@ -286,43 +282,46 @@ EVENT-MSG is a string representation of the event."
...
-    (event . ,event)))
+(defun org-wild-notifier--gather-info (marker)
+  "Collect information about an event.
+MARKER acts like event's identifier."
+  `((times . (,(org-wild-notifier--extract-time marker)))
+    (title . ,(org-wild-notifier--extract-title marker))
+    (intervals . ,(org-wild-notifier--extract-notication-intervals marker))))
 
 (defun org-wild-notifier--stop ()
-  "Stops the notification timer and cancel any in-progress checks."
```
Comment (akhramov) [inline org-wild-notifier.el:321]: this doesn't look right

**Comment by nshi** [comment]
The timer changes should be independent from this but fine to revert if that makes things easier.

File: org-wild-notifier.el (hunk 1 above)
Comment (nshi) [reply org-wild-notifier.el:321]: Hmm, I don't think I changed this in the original commits.

**Pull request merged**